packages/client/components/EmailPasswordAuthForm.tsx

```diff
@@ -90,11 +90,12 @@ const EmailPasswordAuthForm = forwardRef((props: Props, ref: any) => {
   const {location} = useRouter()
   const params = new URLSearchParams(location.search)
   const isSSODefault = Boolean(params.get('sso'))
```

Comment:
+1 When passing `sso=1` like `http://localhost:3000/?sso=1` then this is `true` and the password field is disabled, even when `AUTH_SSO_DISABLED=true`. Maybe change it to
```
  const isSSODefault = process.env.AUTH_SSO_ENABLED && Boolean(params.get('sso'))
```